packages/next-codemod/bin/upgrade.ts

```diff
@@ -600,3 +617,119 @@ function writeOverridesField(
     }
   }
 }
+
+function getDependenciesOutOfRange(
+  appPackageJson: any,
+  versionMapping: Record<string, { version: string; required: boolean }>,
+  allDependenciesToInstall: Record<string, string>
+) {
+  const versionMappingKeys = Object.keys(versionMapping)
+  const dependenciesOutOfRange = new Map<
+    string,
+    { current: string; expected: string; from: string }
+  >()
+
+  if ('peerDependencies' in appPackageJson) {
```

Comment:
Let's leave this out for now and fix in a follow-up. The wording for the warning will need to be different.

Reply:
Hmm yeah, good point. I cannot think of an app that requires `peerDependencies`. I'm good to remove it!

Reply:
Removed.